### PR TITLE
Added CubeMap Capture component documentation

### PR DESCRIPTION
**Pull request by dmcdiarmid-ly**
Signed-off-by: dmcdiarmid-ly <63674186+dmcdiarmid-ly@users.noreply.github.com>

## Change summary

Added CubeMap Capture component documentation.

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/components/reference/atom/cubemap-capture.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+![cubemap-capture-component-base-properties](/images/user-guide/components/reference/atom/cubemap-capture/cubemap-capture-base-properties-ui.png)
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Capture CubeMap** | Button to begin the capture process and generate the cubemap at the entity position.
+| **Capture Type** | Specifies the type of cubemap to be generated. | `Specular IBL`, `Diffuse IBL` | `Specular IBL` |
+| **Specular IBL CubeMap Quality** | Specifies the quality level of the Specular IBL cubemap.  Only visible when **Capture Type** is set to Specular IBL. | `Very Low`, `Low`, `Medium`, `High`, `Very High` | `Medium` |
+| **CubeMap Path** | Displays the output cubemap path and file name.  This is a read-only field for information purposes only.
+| **Exposure** | Specifies the exposure to use when generating the cubemap. | -16.0 to 16.0 | `0.0` |
```
Comment (willihay) [inline content/docs/user-guide/components/reference/atom/cubemap-capture.md:25]:
```suggestion
| **Exposure** | Specifies the exposure to use when generating the cubemap. | `-16.0` to `16.0` | `0.0` |
```

File: content/docs/user-guide/components/reference/atom/cubemap-capture.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: CubeMap Capture Component
+linktitle: CubeMap Capture
+description: 'Open 3D Engine (O3DE) Mesh component reference.'
```
[the remark this answers is not in the document]
Comment (dmcdiarmid-ly) [reply content/docs/user-guide/components/reference/atom/cubemap-capture.md]: Updated.

File: content/docs/user-guide/components/reference/atom/cubemap-capture.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+toc: true
+---
+
+The **CubeMap Capture** component allows a Specular IBL or Diffuse IBL cubemap to be captured at a specific location in the scene.  The resulting cubemap can be used with the **Global Skylight (IBL)** and **Reflection Probe** components.
+
+## Provider ##
```
[the remark this answers is not in the document]
Comment (dmcdiarmid-ly) [reply content/docs/user-guide/components/reference/atom/cubemap-capture.md]: Updated.

File: content/docs/user-guide/components/reference/atom/cubemap-capture.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+
+
+## CubeMap Capture Component properties
```
[the remark this answers is not in the document]
Comment (dmcdiarmid-ly) [reply content/docs/user-guide/components/reference/atom/cubemap-capture.md]: Updated.

File: content/docs/user-guide/components/reference/atom/cubemap-capture.md (hunk 1 above)
Comment (dmcdiarmid-ly) [reply content/docs/user-guide/components/reference/atom/cubemap-capture.md]: Updated.

**Review by willihay — APPROVED** [review]

File: content/docs/user-guide/components/reference/atom/cubemap-capture.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+title: CubeMap Capture Component
+linktitle: CubeMap Capture
+description: Captures a Specular IBL or Diffuse IBL cubemap at the entity location.
+toc: true
+---
+
+The **CubeMap Capture** component allows a Specular IBL or Diffuse IBL cubemap to be captured at a specific location in the scene.  The resulting cubemap can be used with the **Global Skylight (IBL)** and **Reflection Probe** components.
```
Comment (chanmosq) [inline content/docs/user-guide/components/reference/atom/cubemap-capture.md:8]:
```suggestion
The **CubeMap Capture** component captures a Specular IBL or Diffuse IBL cubemap at a specific location in the scene. You can use the resulting cubemap with the **Global Skylight (IBL)** and **Reflection Probe** components.
```

File: content/docs/user-guide/components/reference/atom/cubemap-capture.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+
+
+## Properties
+
+![cubemap-capture-component-base-properties](/images/user-guide/components/reference/atom/cubemap-capture/cubemap-capture-base-properties-ui.png)
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Capture CubeMap** | Button to begin the capture process and generate the cubemap at the entity position.
```
Comment (chanmosq) [inline content/docs/user-guide/components/reference/atom/cubemap-capture.md:21]:
```suggestion
| **Capture CubeMap** | Button to begin the capture process and generate the cubemap at the entity position. |  |  |
```

File: content/docs/user-guide/components/reference/atom/cubemap-capture.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+| **Capture Type** | Specifies the type of cubemap to be generated. | `Specular IBL`, `Diffuse IBL` | `Specular IBL` |
+| **Specular IBL CubeMap Quality** | Specifies the quality level of the Specular IBL cubemap.  Only visible when **Capture Type** is set to Specular IBL. | `Very Low`, `Low`, `Medium`, `High`, `Very High` | `Medium` |
```
Comment (chanmosq) [inline content/docs/user-guide/components/reference/atom/cubemap-capture.md:23]:
```suggestion
| **Specular IBL CubeMap Quality** | Specifies the quality level of the Specular IBL cubemap.  Only visible when **Capture Type** is set to `Specular IBL`. | `Very Low`, `Low`, `Medium`, `High`, `Very High` | `Medium` |
```

**Review by chanmosq — APPROVED** [review]
Looks good, minor suggestions to switch to active voice

**Pull request merged**